www/source/partials/docs/_glossary-launcher.html.md.erb

```diff
@@ -0,0 +1,23 @@
+# <a name="glossary-launcher" id="glossary-launcher" data-magellan-target="glossary-launcher">Launcher</a>
+
+Habitat's Launcher is a sidecar process for the Supervisor which provides a mechanism for launching processes on behalf of the Supervisor. It is the entry point for running the Supervisor and is the Supervisor for the Supervisor. Whereas the Supervisor is able to automatically update itself, the Launcher is currently released a bit differently, by design; it should be rare that the Launcher ever needs to change.
+
+To update your Launchers, run:
+
+```shell
+hab pkg install core/hab-launcher
+```
+
+Then restart the Supervisor. This will, by necessity, require a restart of supervised services, so factor that into your planning.
+
+The Launcher is designed to run as process 1; it is minimal by design. Its responsibilities are to:
+
+* Launch the Habitat Supervisor
+* Restart the Habitat Supervisor on shutdown (Supervisor for the Supervisor)
+* Start, stop, and restart processes on behalf of the Supervisor
```

Comment:
This is changing, starting with https://github.com/habitat-sh/habitat/pull/6107 (the Launcher will only start processes). More functionality is being pulled into the Supervisor; eventually the Launcher is going to be little more than the parent process of everything.

On this third point, I'd probably just say that the Launcher serves as the parent process of the services, which conveys the same intent, but without coupling it so tightly to implementation details.